src/main/java/com/pingcap/tikv/operation/KVErrorHandler.java

```diff
@@ -64,7 +66,11 @@ public void handle(RespT resp) {
             ctxRegion.getLeader().getStoreId()));
         long newStoreId = error.getNotLeader().getLeader().getStoreId();
         regionManager.updateLeader(ctxRegion.getId(), newStoreId);
-
+        regionManager.incrementCacheAccumulator(
```

Comment:
Name it record event might be better since you actually don't want to introduce dependency to Spark accumulator.

Reply:
UR right, will change it later.